src/lean_spec/subspecs/forkchoice/store.py

```diff
@@ -261,23 +261,57 @@ def on_attestation(
             }
         )
 
-    def _validate_block_signatures(self, block: Block, signatures: BlockSignatures) -> bool:
+    def _validate_block_signatures(
+        self,
+        block: Block,
+        signatures: BlockSignatures,
+        proposer_attestation: Attestation,
+    ) -> bool:
         """
         Validate block signatures.
 
-        Temporary stub for aggregated signature validation.
-
         Args:
             block: Block to validate.
             signatures: Block signatures to validate.
+            proposer_attestation: Attestation from the block proposer.
```

Comment:
Nit: just for consistency. Also rearrange if above suggestion is accepted

```suggestion
            proposer_attestation: Proposer's attestation to validate.
```